addons/medical_treatment/initSettings.sqf

```diff
@@ -25,6 +25,24 @@
     true
 ] call CBA_fnc_addSetting;
 
+[
+    QGVAR(bandageRollover),
+    "CHECKBOX",
+    [LSTRING(bandageRollover_DisplayName), LSTRING(bandageRollover_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory_Treatment)],
+    true,
+    false // server can force if necessary, otherwise client decides
```

Comment:
I don't understand why, this means everyone can decide how _effective_ their bandages are, that doesn't make sense to me.

Reply:
It's a tradeoff, bandage slower in exchange for not wasting your bandage's effectiveness (if you're even bandaging more than one kind of wound at once, which is relatively uncommon). You're not getting more effectiveness than whatever's in the config, overall.

I see it as no different as choosing to only use my Elastic Bandages on Large Avulsions or wounds with count > 1.

Reply:
Aah that makes sense. So if I leave this as default it will just take longer to bandage when it will rollover, and if I disable it rollover won't happen but bandaging would be faster.